clusters/app.ci/registry-access/cnv-qe/admin_rbac.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Test infrastructure that supports cnv-qe
+    openshift.io/display-name: cnv-qe CI
+  name: cnv-qe
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: image-puller
+  namespace: cnv-qe
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cnv-qe-image-puller-binding
+  namespace: ocp
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-puller
+subjects:
+- kind: ServiceAccount
+  namespace: cnv-qe
+  name: image-puller
+---
+kind: Group
+apiVersion: user.openshift.io/v1
+metadata:
+  name: cnv-qe-admins
+users:
+- sarahbx
+- dollierp
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cnv-qe-admins-binding
+  namespace: cnv-qe
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: admin
```

Comment:
Please use `view` and `pull-secret-namespace-manager` as documented https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/

Reply:
Updated according to the instructions of the document.